src/util/substrateProviders.ts

```diff
@@ -52,8 +65,32 @@ export const customTypes: CustomTypes = {
   [CHAIN_2]: types[CHAIN_2]
 };
 
+export const bridgeIds: GenericKeyStringConfigs = {
+  [CHAIN_1]: 'rlto',
```

Comment:
This doesn't look very nice with `CHAIN_1` & `CHAIN_2` environment varaiables. It's assuming `CHAIN_1` is Rialto anyway, so it doesn't make sense to pretend it can be generic.